### ISSUE TITLE
Error while installing i2cssh

### ISSUE THREAD

**Issue by yogeshmulay** [wouterdebie/homebrew-repo#71]
MAC-OS 14.4 (23E214)
python3 --version  :  Python 3.12.2

pip3 --version. :  pip 24.0 from /opt/homebrew/lib/python3.12/site-packages/pip (python 3.12)

Getting the following error any suggestions ?

==> Installing i2cssh from wouterdebie/repo
==> python3 -m venv --system-site-packages --without-pip /opt/homebrew/Cellar/i2cssh/0.1.9/libexec
==> python3 -m pip --python=/opt/homebrew/Cellar/i2cssh/0.1.9/libexec/bin/python install /private/tmp/i2cssh--click-20240312-67893-b9q31i/click-8.1.6
==> python3 -m pip --python=/opt/homebrew/Cellar/i2cssh/0.1.9/libexec/bin/python install /private/tmp/i2cssh--click-option-group-20240312-67893-ohlhgh/click-option-group-0.5.6
==> python3 -m pip --python=/opt/homebrew/Cellar/i2cssh/0.1.9/libexec/bin/python install /private/tmp/i2cssh--iterm2-20240312-67893-xskypx/iterm2-2.6
==> python3 -m pip --python=/opt/homebrew/Cellar/i2cssh/0.1.9/libexec/bin/python install /private/tmp/i2cssh--protobuf-20240312-67893-5ypynt/protobuf-4.24.1
Last 15 lines from /Users/ymulay200@dx.com/Library/Logs/Homebrew/i2cssh/05.python3:
  3 errors generated.
  error: command '/opt/homebrew/Library/Homebrew/shims/mac/super/clang' failed with exit code 1
  error: subprocess-exited-with-error

  × Building wheel for protobuf (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> See above for output.

  note: This error originates from a subprocess, and is likely not a problem with pip.
  full command: /opt/homebrew/Cellar/i2cssh/0.1.9/libexec/bin/python /opt/homebrew/lib/python3.12/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py build_wheel /private/tmp/tmp9hx_tbwv
  cwd: /private/tmp/i2cssh--protobuf-20240312-67893-5ypynt/protobuf-4.24.1
  Building wheel for protobuf (pyproject.toml): finished with status 'error'
  ERROR: Failed building wheel for protobuf
Failed to build protobuf
ERROR: Could not build wheels for protobuf, which is required to install pyproject.toml-based projects

If reporting this issue please do so at (not Homebrew/brew or Homebrew/homebrew-core):
  https://github.com/wouterdebie/homebrew-repo/issues

K202924 ~ %

**Comment by chrisbergeron** [comment]
I'm getting the exact same error:

```bash
==> Installing i2cssh from wouterdebie/repo
==> python3 -m venv --system-site-packages --without-pip /opt/homebrew/Cellar/i2cssh/0.1.9/libexec
==> python3 -m pip --python=/opt/homebrew/Cellar/i2cssh/0.1.9/libexec/bin/python install /private/tmp/i2cssh--click-20240318-19677-lpzs1r/click-8.1.6
==> python3 -m pip --python=/opt/homebrew/Cellar/i2cssh/0.1.9/libexec/bin/python install /private/tmp/i2cssh--click-option-group-20240318-19677-ujr5xj/click-option-group-0.5.6
==> python3 -m pip --python=/opt/homebrew/Cellar/i2cssh/0.1.9/libexec/bin/python install /private/tmp/i2cssh--iterm2-20240318-19677-26bj22/iterm2-2.6
==> python3 -m pip --python=/opt/homebrew/Cellar/i2cssh/0.1.9/libexec/bin/python install /private/tmp/i2cssh--protobuf-20240318-19677-isqab/protobuf-4.24.1
Last 15 lines from /Users/cbergeron/Library/Logs/Homebrew/i2cssh/05.python3:
  3 errors generated.
  error: command '/opt/homebrew/Library/Homebrew/shims/mac/super/clang' failed with exit code 1
  error: subprocess-exited-with-error

  × Building wheel for protobuf (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> See above for output.

  note: This error originates from a subprocess, and is likely not a problem with pip.
  full command: /opt/homebrew/Cellar/i2cssh/0.1.9/libexec/bin/python /opt/homebrew/lib/python3.12/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py build_wheel /private/tmp/tmpmz483lxg
  cwd: /private/tmp/i2cssh--protobuf-20240318-19677-isqab/protobuf-4.24.1
  Building wheel for protobuf (pyproject.toml): finished with status 'error'
  ERROR: Failed building wheel for protobuf
Failed to build protobuf
ERROR: Could not build wheels for protobuf, which is required to install pyproject.toml-based projects

If reporting this issue please do so at (not Homebrew/brew or Homebrew/homebrew-core):
  https://github.com/wouterdebie/homebrew-repo/issues

These open issues may also help:
Error while installing i2cssh https://github.com/wouterdebie/homebrew-repo/issues/71

$ which python3
/opt/homebrew/bin/python3

$ python3 --version
Python 3.12.2

$ pip3 --version
pip 24.0 from /opt/homebrew/lib/python3.12/site-packages/pip (python 3.12)
```

**Comment by wouterdebie** [comment]
This should have been fixed now in 0.1.10 and higher. Let me know if it still doesn't work.
